WBooks/Library.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Library.swift
+//  WBooks
+//
+//  Created by ana.mancuso on 20/09/2022.
+//
+
+import UIKit
```

Comment:
for the creation of this class it is not necessary to use anything from UiKit, so it would not be necessary to import it

Reply:
I'm using UIImage Therefore it tells me that I have to use UIKit